ssz/src/lib.rs

```diff
@@ -34,11 +34,14 @@
 //!
 //! See `examples/` for manual implementations of the `Encode` and `Decode` traits.
 
+mod bitfield;
 mod decode;
 mod encode;
 pub mod legacy;
 mod union_selector;
 
+#[doc(hidden)]
```

Comment:
I've hidden these types in the docs which I believe means the current documentation should remain unchanged.

Since we are re-exporting the Bitfield types in `ssz_types` the documentation for them will still exist in that crate. Like it does now.

There might be an argument for moving the documentation across (or duplicating it), but I'm loosely in favour of this approach for now.